pyproject.toml

```diff
@@ -35,7 +35,8 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-graphql-core = "~3.2.0"
+# TOOD: revert this
+graphql-core = {git = "https://github.com/graphql-python/graphql-core.git", branch = "main"}
```

Comment:
In case we want to merge this before an official graphql-core relase, we should pin the commit in which your changes landed.

Reply:
could we conditionally support it based on the version of GraphQL core?

Reply:
Strawberry tests won't pass 3.3
https://github.com/strawberry-graphql/strawberry/actions/runs/9928799671/job/27425551808?pr=3570#step:6:7838

I think it is possible to support it based on the version but I am not sure it would be the place to add support for 3.3. maybe in another PR.